tests/testlib/Makefile

```diff
@@ -27,4 +29,4 @@ libtests2n.so: ${OBJS}
 	${CC} ${CFLAGS} -shared ${LDFLAGS} -o libtests2n.so ${OBJS}
 
 libtests2n.dylib: ${OBJS}
-	test ! -f /usr/lib/libSystem.dylib || libtool ${LDFLAGS} -dynamic -o libtests2n.dylib ${OBJS}
+	${CC} ${ARCHFLAGS} ${LDFLAGS} -dynamiclib -o libtests2n.dylib ${OBJS}
```

Comment:
I don't think so, I've never seen it elsewhere. The test is a check for "are we on OS X". Any other check would work too, maybe uname?